ksml-runner/src/main/java/io/axual/ksml/runner/config/NotationConfig.java

```diff
@@ -0,0 +1,37 @@
+package io.axual.ksml.runner.config;
+
+/*-
+ * ========================LICENSE_START=================================
+ * KSML Runner
+ * %%
+ * Copyright (C) 2021 - 2023 Axual B.V.
+ * %%
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * =========================LICENSE_END==================================
+ */
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import lombok.Builder;
+import lombok.Data;
+import lombok.extern.jackson.Jacksonized;
+
+import java.util.Map;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+@Builder
+@Jacksonized
+@Data
+public class NotationConfig {
```

Comment:
Should this be mutable?
I think all of this can be  a `record` instead of a `class`

Reply:
Converted into a record